blogo-input/posts/evoting-review.md

```diff
@@ -0,0 +1,100 @@
+    
+ * Token voting: In the context of web3, elections may be based on, e.g. NFT ownership. Here the census should be implicitly linked to the NFT ownership data stored on Ethereum.
+    
+* Hiding voting power: In voting based on NFT ownership, a desirable property is that it should not be possible to infer whether a given voter owns a large amount of tokens, hence "whales" should not be identifiable. In practice it may be known that Bob is the major token holder so knowing that a particular ballot was cast by someone with a very high amount of tokens implies that the relevant ballot was likely cast by Bob.
+  
+ * Fairness: E-voting in the web3 domain is usually performed using proofs of membership as described before. If e-voting protocols based on simple membership proofs, voter preferences sent to the smart contract are publicly visible. This poses the problem of fairness: the voters who have not yet cast a ballot can be influenced by the partial tally that is publicly visible on the blockchain.
+    
+ * Equity of costs: The gas fees are paid by whoever submits a transaction and this has consequences. In traditional e-voting systems, computing a tally at the end of the election carries a negligible cost. In Ethereum, however, one has to prefer solutions in which the cost of tallying the result is fairly divided among the voters. If, for instance, a system were such that the last voter needs to compute the tally on-chain in an expensive way, his transaction would cost more gas than for previous voters and this would raise an equity issue.
+    
+ * Storage: The cost of storage in Ethereum is very high. Preference must therefore be given to solutions in which only a short digest is published on-chain,i.e. on Ethereum, and the full data is stored off-chain.
+
+    The cost of storage also poses some equity problems. In a system where each voter needs to add an element to a set, if this cost grows linearly with the number of elements in the set and the number of voters is very high ($> 10k$), the last voters would pay more than the first voters.
+    
+    ## E-voting projects and the Nouns DAO call
+    
+    We defer to the [review](https://github.com/aragonzkresearch/blog/blob/main/pdf/evoting_review.pdf) for a list of recent e-voting projects including one by [Aragon ZK Research](https://research.aragon.org/) and by the [VocDoni](https://vocdoni.io/) guild. The list has not be meant to be exhaustive and there are other ongoing projects. Moreover, we gave preference to the recent [Nouns DAO]({https://prop.house/nouns/private-voting-research-sprint}) call to which we of Aragon ZK Research participated along with Aztec. This is ongoing research and we invite you to stay updated on our blog to see the next progress in the coming months!
+    ## Acknowledgements
```

Comment:
I don't know if this is intentional, but the `Acknowledgements` section is tabulated and seems to be a subsection of `E-voting in the web3 domain`.

Same for `References` section.